src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -1160,7 +1160,7 @@ elementFormDefault="qualified">
     </xs:element>
     <xs:element name="Folder" type="msb:SimpleItemType" substitutionGroup="msb:Item">
         <xs:annotation>
-            <xs:documentation><!-- _locID_text="Folder" _locComment="" -->Folder on disk</xs:documentation>
+            <xs:documentation><!-- _locID_text="Folder" _locComment="" -->Used mostly by Visual Studio to identify an empty folder.</xs:documentation>
```

Comment:
nit. remove "mostly" to avoid unnecessary insinuation.

Reply:
I wasn't sure it's used only by VS and I didn't want to lie :D But it makes sense... Removed.